generate_dependency_graph_dot.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import pygraphviz as pgv
+from modelrecords.repository import Repository
+
+styles = {
+    'family': dict(shape="house", style="rounded, filled", penwidth=0.5, height=0.25, fillcolor="#CBC3E3", fontname="Helvetica"),
+    'dataset': dict(shape="cylinder", style="rounded, filled", penwidth=0.5, width=0.75, height=0.25, fillcolor="#E0B7F3", fontname="Helvetica"),
+    'model': dict(shape="box", style="rounded, filled", penwidth=0.5, height=0.25, fontcolor="#FFF", fillcolor="#800080", fontname="Helvetica")
+}
+
+
+def render_modelgraph(nodes, edges, records):
```

Comment:
This seems to be duplicated in `dependency_graph.py`

Reply:
They are very similar, perhaps we can consolidate this logic since I suspect we will want to iterate on it down the line.